Recast/Source/Recast.cpp

```diff
@@ -71,6 +71,56 @@ void rcContext::log(const rcLogCategory category, const char* format, ...)
 	doLog(category, msg, len);
 }
 
+rcAreaModification::rcAreaModification() :
+	m_value(RC_WALKABLE_AREA),
+	m_mask(RC_WALKABLE_AREA)
+{
+}
+
+rcAreaModification::rcAreaModification(unsigned char value) :
+	m_value(value),
+	m_mask(RC_WALKABLE_AREA)
```

Comment:
This mask should be 0xFF (for now, it will change in future).
`RC_WALKABLE_AREA` has no logical significance here; its value being 63 is simply by chance.